Build is failing

While trying to build pikaur itself I am getting:

```
==> Starting build()...
# find pikaur -type f -name '*.py' -not -name 'argparse.py' \
	#
python ./maintenance_scripts/pikaman.py README.md pikaur.1
make: python: Permission denied
make: *** [Makefile:37: pikaur.1] Error 127
make: *** Waiting for unfinished jobs....
find pikaur -type f -name '*.py' \
	| xargs xgettext --language=python --add-comments --sort-output \
		--default-domain=pikaur --from-code=UTF-8 --keyword='_n:1,2' --output=locale/pikaur.pot
==> ERROR: A failure occurred in build().
```

--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ POTEMPFILES := $(addprefix $(LOCALEDIR)/,$(addsuffix .po~,$(LANGS)))
 MOFILES = $(POFILES:.po=.mo)
 DISTDIR := dist
 
-PIKAMAN := python ./maintenance_scripts/pikaman.py
+PIKAMAN := $(shell which python) ./maintenance_scripts/pikaman.py
 README_FILE := README.md
 MAN_FILE := pikaur.1
 

--- a/Makefile
+++ b/Makefile
@@ -9,6 +9,8 @@ POTEMPFILES := $(addprefix $(LOCALEDIR)/,$(addsuffix .po~,$(LANGS)))
 MOFILES = $(POFILES:.po=.mo)
 DISTDIR := dist
 
+# A $(which python) is necessary here to avoid a bug in make that would
+# try to execute a directory named "python". See https://savannah.gnu.org/bugs/?57962
 PIKAMAN := $(shell which python) ./maintenance_scripts/pikaman.py
 README_FILE := README.md
 MAN_FILE := pikaur.1

